content/learning-paths/servers-and-cloud-computing/cca-veraison/attestation-verification.md

```diff
@@ -0,0 +1,97 @@
+
+The URL for reaching this experimental verifier service is http://veraison.test.linaro.org:8080
+
+To check that you can reach the Linaro attestation verifier service, run the following command:
+
+```bash
+curl http://veraison.test.linaro.org:8080/.well-known/veraison/verification
+```
+
+This is a simple call to query the well-known characteristics of the verification service. If it succeeds, it will return a JSON response that looks something like this:
+
+```output
+{
+  "ear-verification-key": {
+    "alg": "ES256",
+    "crv": "P-256",
+    "kty": "EC",
+    "x": "usWxHK2PmfnHKwXPS54m0kTcGJ90UiglWiGahtagnv8",
+    "y": "IBOL-C3BttVivg-lSreASjpkttcsz-1rb7btKLv8EX4"
+  },
+  "media-types": [
+    "application/vnd.parallaxsecond.key-attestation.tpm",
+    "application/eat-cwt; profile=\http://arm.com/psa/2.0.0\",
+    "application/eat+cwt; eat_profile=\"tag:psacertified.org,2023:psa#tfm\"",
+    "application/eat-collection; profile=\http://arm.com/CCA-SSD/1.0.0\",
+    "application/eat+cwt; eat_profile=\"tag:psacertified.org,2019:psa#legacy\"",
+    "application/vnd.enacttrust.tpm-evidence",
+    "application/vnd.parallaxsecond.key-attestation.cca",
+    "application/psa-attestation-token",
+    "application/pem-certificate-chain"
+  ],
+  "version": "commit-2063e7e",
+  "service-state": "READY",
+  "api-endpoints": {
+    "newChallengeResponseSession": "/challenge-response/v1/newSession"
+  }
+}
+```
+
+This JSON response contains all the information that you need to use the verification service. Review the different JSON properties.
+
+- The `ear-verification-key` is the cryptographic key that you will use later to verify the results that are returned by the service.
+
+- The `media-types` entry provides the list of the different attestation data formats that the verification service supports. If you look down this list, you will find an entry for the CCA profile of the EAT format. It is the fourth entry in the list. This tells us that the service is capable of processing Arm CCA attestation tokens.
+
+- The `api-endpoints` entry describes the set of RESTful APIs that are supported by the service. When verifying an attestation token, you will use the challenge-response API.
+
+If you can reach the verification service, you are now ready to use it to evaluate the CCA example token.
+
+## Save the Public Key of the Verification Service
+
+One of the properties that was returned in the previous step was the public key of the verification service. This key will be needed later to check the signature on the attestation results. All that is needed in this step is to copy the contents of the `ear-verification-key` field from the previous step and save it to a separate JSON file.
+
+The easiest way to do this is to use the jq utility, which is a popular command-line tool that can be used to parse and manipulate JSON data. You can install it using your local package manager, for instance:
```

Comment:
We should probably move this install step to the prerequisites section (just like the Git and Go installations)

Reply:
moved